guides/common/modules/con_shellhooks-plugin.adoc

```diff
@@ -0,0 +1,20 @@
+[id="shellhooks_{context}"]
+= Shellhooks
+
+With webhooks one {Project} event can only be mapped to one API call.
+For advanced integrations, where a single shell script can contain multiple commands, you can install a {SmartProxy} shellhooks plugin that exposes executables using a REST HTTPS API.
+
+A webhook can then be configured to reach out to a {SmartProxy} API to run a predefined shellhook, which can contain commands or can edit files.
```

Comment:
which can `e.g.` contain commands or can edit files.

These are just two examples, but of course the shell script can do whatever shell can do.

Reply:
Done.